src/main/resources/swagger/ga4gh-tool-discovery.yaml

```diff
@@ -268,6 +268,12 @@ definitions:
       descriptor:
         type: string
         description: The descriptor that represents this version of the tool. (CWL or WDL)
+      format: 
+        description: The output type of the descriptor. If not specified it is up to the underlying implementation to determine which output format to return.
```

Comment:
The second sentence here ("If not specified...") makes sense in the context of query parameter, but not field description, right?